### ISSUE TITLE
On a clean checkout of the repo, npm run test:cover doesn't create any files

### ISSUE THREAD

**Issue by rstudner** [coryhouse/react-slingshot#277]
```
git clone git@github.com:coryhouse/react-slingshot.git
cd react-slingshot
npm install
npm run test:cover
```

This leaves me with a coverage folder, with nothing inside of it.

Is this expected?

**Comment by coryhouse** [comment]
Hrmm, that's unfortunate. Yes, I'm seeing this too, so something has regressed. I'm currently buried on a late project but would welcome a pull request from anyone with the time to debug this. 

Thanks for the heads up!

**Comment by marinbcaro** [comment]
It looks like Isparta has already an open issue about this: https://github.com/douglasduteil/isparta/issues/126
I did what they suggested: adding "istanbul": "0.4.4" to package.json and that worked for me. @coryhouse should I create a pull request for this or wait until the issue is fixed by Isparta?

**Comment by coryhouse** [comment]
Fixed in #294. Thanks @marinbcaro!
